pkgbot/__init__.py

```diff
@@ -269,7 +269,7 @@ def process_request(data):
     # everything copied, run aptly-commands
     aptly_pub = []
     for repo, pkg_path in aptly_add:
-        os.write(fifo, "aptly repo add {0} {1}\n".format(repo, pkg_path))
+        os.write(fifo, "aptly repo add {0} {1}\n".format(repo, pkg_path).encode('ASCII'))
```

Comment:
Why ASCII? Is there a restriction from aptly that requires ASCII? If not, I'd stick to the default (UTF-8) to avoid surprises:

```suggestion
        os.write(fifo, "aptly repo add {0} {1}\n".format(repo, pkg_path).encode())
```